### PR TITLE
getPlayerFromSteamId64 method

### PR DESCRIPTION
**Pull request by saddiqs1**
getPlayerFromSteamId64 method, fixed typescript/linting errors, updated readme

### CODE REVIEW

**Comment by saddiqs1** [comment]
Thinking about it, the code could probably be cleaned up a little more to make the calls a bit more modular and clear for when future calls are added, i.e. moving `fetch(hero: Omit<Hero, 'then'>, url: string): Promise<any>` from `player.ts` to a generic helper file somewhere, and then having `player.ts` purely responsible for retrieving information related to a players esea profile. You could then make a seperate `teams.ts` file for example in the future which contains calls to retrieve team info

File: src/player.ts [hunk 1]
```diff
@@ -31,19 +34,19 @@ export async function getPlayer(
...
+    const statsUrl = `${origin}/users/${eseaProfileId}/stats?filters[type_scopes]=pug&filters[period_types]=career`;
+    const lastMatchUrl = `${origin}/users/${eseaProfileId}/matches?page_size=1`;
 
     this.debug(`Going to ${origin}`);
     const originResponse = await hero.goto(origin, { timeoutMs: this.timeout });
-    const statusCode = originResponse.response.statusCode;
+    const { statusCode } = originResponse.response;
     if (statusCode !== 200) {
       // Check for cloudflare challenge
-      const title = await hero.document.querySelector("title");
```
Comment (sruusk) [inline src/player.ts:45]: I think this call needs to be awaited due to the way ulixee hero functions.

File: src/player.ts [hunk 2]
```diff
@@ -31,19 +34,19 @@ export async function getPlayer(
...
-      if(title != undefined && await title.textContent === "Attention Required! | Cloudflare"){
```
Comment (sruusk) [inline src/player.ts:46]: This call needs to be awaited due to how the [AwaitedDOM ](https://ulixee.org/docs/hero/basic-client/awaited-dom) works in ulixee hero.

**Comment by sruusk** [comment]
Thanks for this. 
The code could certainly be cleaned a lot, but I currently have no intention of building this much further than it is currently.
But if you need some features added fell free to open a pull request. :)

**Pull request merged**